README.md

```diff
@@ -28,14 +28,23 @@ planx-new is a monorepo containing our full application stack. Here's a quick su
 
 1. Run the following command to get everything (postgres, sharedb, api and hasura server processes) up and running `docker-compose up --build -d`
 
-1. Move into the hasura directory `cd ../hasura.planx.uk` & install dependencies `pnpm i`
+1. Move into the hasura directory `cd ../hasura.planx.uk` & install dependencies `pnpm i`. 
 
 1. Open [Hasura's](https://hasura.io/) web console `pnpm start` & check that your Google email address is in the `users` table, if not then add it
 
-1. Move into the editor directory `cd ../editor.planx.uk` & install dependencies `pnpm i`
+1. Move into the editor directory `cd ../editor.planx.uk` & install dependencies `pnpm i`. 
 
 1. Start the dev server! `pnpm start` & open http://localhost:3000 & login with your GMail/Google email address
 
+
+### Troubleshooting
+
+If you run into trouble, you may want to try the following:
+
+* Ensure you have a local `.env` file with up-to-date keys etc (see relevant `.env.example` files for reference and 1Password for an initial `.env` file).
+* The Hasura CLI may require a postinstall step which can be run manually with `cd node_modules/hasura-cli && pnpm postinstall`.
```

Comment:
You're the first new dev to run through these steps in full since we upgraded to pnpm 7 - could you try toggling `side-effects-cache` and see if it makes the postinstall script run?

https://github.com/pnpm/pnpm/issues/4649

Docs: https://pnpm.io/npmrc#side-effects-cache

Reply:
Ah, yes, I found the culprit. I have `ignore-scripts=true` in my ~/.npmrc like one of the commenters in that issue thread.

Thanks for helping identify the issue here.